style.css

```diff
@@ -484,3 +503,331 @@ textarea {
   margin-right: auto;
   border-radius: 10px;
 }
+
+@media (min-width: 768px) {
+  body {
+    width: 768px;
+  }
+
+  .nodisplay-desktop {
+    display: none;
+  }
+
+  .nonvisible {
+    visibility: hidden;
+  }
+
+  .navbar ul {
+    display: inline;
+    display: flex;
+    width: 295px;
+    align-content: center;
+    justify-content: space-between;
+    position: relative;
+    bottom: 12px;
+    margin-right: 130px;
+  }
+
+  .card1 a {
+    display: inline-block;
+  }
+
+  .navbar ul a {
+    color: #fff;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .socials li {
+    margin-bottom: 18px;
+  }
+
+  .navbar ul li {
+    list-style: none;
+  }
+
+  .navbar {
+    padding-left: 130px;
+    padding-top: 24px;
+  }
+
+  .main {
+    background-image: none;
+    height: 798px;
+    margin-left: 143px;
+    width: 566px;
+    top: 264px;
+  }
+
+  .main h2 {
+    font-family: 'Crete Round', sans-serif;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .main h3 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .main p {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  #first-page {
+    background-image: url("images/desktop-bg.png");
+    background-position: right top;
+    background-repeat: no-repeat;
+    height: 870px;
+    width: 1440px;
+  }
```

Comment:
- [ ] As I said in the previous comment, try to not fix the width of the layouts and use so your page fits all the desktop screen sizes. 
- [ ] Kindly work more on the width of the layouts in `works` section to improve your UI design:
<img width="958" alt="image" src="https://user-images.githubusercontent.com/77190203/161115228-eb2ca354-83ef-4dd4-ae85-36c12e375c25.png">